packages/formbuilder/src/definitions.js

```diff
@@ -110,7 +110,7 @@ const basicInfo: FieldDef[] = [
     name: "name",
     type: "text",
     required: true,
-    defaultValue: "",
+    defaultValue: " ", // TODO: investigate where it fails
```

Comment:
AkField requires name property, couldn't figure out where it fails.

Reply:
Awesome thanks.